working-open.md

```diff
@@ -0,0 +1,53 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that, we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share and provide opportunities for self- and collective- learning as we work in solidarity with others seeking to create meaningful livelihoods.
+- Not because "default to open" is an ideal in itself, but because it orients us toward negotiating a set of practices around our shared intentions.
+- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay out our practices of consent and privacy
+
+# How and what?
+
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
+
+Legend:  
+👀  view only  
+📝  view and participate  
+
+**Default open to all**:
+
+- Scope: Organization
+  - Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀
+  - Aggregated financal statements 👀
+- Scope: Project
+  - Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀
+
+* *This would include source repositories and code (e.g., on GitHub) and other process materials based on consensus on licensing and usage constraints.*
```

Comment:
```suggestion
* *This would include source repositories and code (e.g. on GitHub) and other process materials based on consensus on licensing and usage constraints.*
```